bottles/frontend/windows/gamescope.py

```diff
@@ -54,6 +58,17 @@ def __init__(self, window, config, **kwargs):
 
         self.__update(config)
 
+        # Set VRR row to visible if the compositor supports VRR
+        if vrr_available:
+            self.row_vrr.set_visible(True)
+
+            if DisplayUtils.display_server_type() == "wayland":
+                warning = _("This feature depends on the version of the compositor.")
+                learn_more = _("Learn more about variable refresh rate")
+                link = f"<a href=\"LINK\">{learn_more}</a>"
```

Comment:
This needs to be updated. @mirkobrombin can we create a page for VRR?

Reply:
I guess this works. https://wiki.archlinux.org/title/Variable_refresh_rate

Reply:
I think ArchWiki cover it better.